src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,13 +48,29 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        Set<String> lowerCaseKeywords = convertStringsToLowerCase(keywords);
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            Set<String> lowercaseWordsInName = convertStringsToLowerCase(wordsInName);
+            if (!Collections.disjoint(lowercaseWordsInName, lowerCaseKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Converts all strings in a set to lowercase
+     *
+     * @param stringsSet
+     * @return set of all strings in lowercase
+     */
+    private Set<String> convertStringsToLowerCase(Set<String> stringsSet) {
+        Set<String> lowerCaseStringsSet = new HashSet<>();
```

Comment:
Why not use `Set` instead of `HashSet`?

Reply:
Err! Ignore this; my miss. :man_facepalming: 
Set is the interface. HashSet is the class implementing it